### ISSUE TITLE
Video played automatically eventhough paused=true

### ISSUE THREAD

**Issue by aluksidadi** [TheWidlarzGroup/react-native-video#592]
Im having weird problem:

I hardcoded the paused value to true, but somehow the video got played automatically

```js
              <Video
                ref={(player) => this.player = player}
                source={{uri: user.user_profile.video}}
                width={width}
                height={height - 160}
                repeat={false}
                paused={true}
                muted={false}
                resizeMode="cover"
              />
```

Anyone experiencing same issue?

**Comment by fogil** [comment]
Yes, I even hardcoded rate to == 0.0 and it still autoplays at rate 1.0.

Only workaround is to not render the Video component at all, until app logic dictates that a video should be played then go ahead and render Video -- after which it will always autoplay no matter what settings are used.

**Comment by aluksidadi** [comment]
I think there was an autoplay bug on 1.0.0.
So I managed to solved this by pulling the package from the latest master branch
(i tried putting version "^1.1.0" on my package.json but seems like the owner hasnt pushed 1.1.0 to npm yet eventhough the package.json in this repository tells me that latest version is 1.1.0)

**Comment by nickkapoor** [comment]
Yes, i see the issue.  Tried with paused as true and rate as 0.  Will test out your recommendation to d/l the 1.1.0 version.

**Comment by BuildWithLal** [comment]
facing same issue on Android.. `"react-native-video": "^1.0.0"`.
@aluksidadi i tried pulling latest version `master` from repo but still facing same issue. Any other changes i am missing?

**Comment by rachoac** [comment]
Any updates on the root cause, or fixing of this issue?

**Comment by aluksidadi** [comment]
@lalzada it fixed my autoplay issue by importing the master branch. but theres actually another bug i found where it kept playing in a loop after being played eventhough loop is false

**Comment by EdenGottlieb** [comment]
I found a workaround, a bit ugly but it works.

Set the "paused" attribute to a property of state that is initially set to false. Then, Set the OnLoad function to change this property to true. 
This works perfectly on Android. and the video is paused upon rendering, no "micro-stuttering for a moment" or something like that. Haven't tried it on iOS.

**Comment by GingerBear** [comment]
@EdenGottlieb 's workaround works as well on iOS with no issue.

**Comment by Symous** [comment]
I just fix this but the author hasn't merge this PR #625 .

**Comment by cobarx** [comment]
I fixed this in #1082. In my testing everything is working properly now. Will be released in 3.0.
